core/templates/dev/head/pages/exploration_editor/simple_editor_tab/questions/MultipleChoiceEditorDirective.js

```diff
@@ -41,6 +41,16 @@ oppia.directive('multipleChoiceEditor', [function() {
         // Note that a questionId generated in this way may contain spaces,
         // since it is just the state name.
         $scope.questionId = $scope.getUniqueId();
+
+        // Returns a unique id for sub-elements of a question given a suffix.
```

Comment:
Can you pull out some of the common functionality for this and the other hashes into a shared service?